tests/data_context/store/test_ge_cloud_store_backend.py

```diff
@@ -44,6 +46,115 @@ def _closure(resource_type: GeCloudRESTResource) -> GeCloudStoreBackend:
     return _closure
 
 
+@pytest.mark.parametrize(
+    "base_url,organization_id,resource_name,id,expected",
+    [
+        pytest.param(
+            "https://app.test.greatexpectations.io"
+            "de5b9ca6-caf7-43c8-a820-5540ec6df9b2",
+            "my_resource",
+            "my_value",
+            None,
+            "https://app.test.greatexpectations.iode5b9ca6-caf7-43c8-a820-5540ec6df9b2/organizations/my_resource/my-value",
+            id="no id",
+        ),
+        pytest.param(
+            "https://app.test.greatexpectations.io"
+            "de5b9ca6-caf7-43c8-a820-5540ec6df9b2",
+            "my_resource",
+            "my_value",
+            "8746e25c-de4d-450d-967b-df0d5546590d",
+            "https://app.test.greatexpectations.iode5b9ca6-caf7-43c8-a820-5540ec6df9b2/organizations/my_resource/my-value/8746e25c-de4d-450d-967b-df0d5546590d",
+            id="with id",
+        ),
+    ],
+)
+@pytest.mark.unit
+@pytest.mark.cloud
+def test_construct_url(
+    base_url: str,
+    organization_id: str,
+    resource_name: str,
+    id: Optional[str],
+    expected: str,
+) -> None:
+    assert (
+        construct_url(
+            base_url=base_url,
+            organization_id=organization_id,
+            resource_name=resource_name,
+            id=id,
+        )
+        == expected
+    )
+
+
+@pytest.mark.parametrize(
+    "resource_type,organization_id,attributes_key,attributes_value,kwargs,expected",
+    [
+        pytest.param(
+            "my_resource",
+            "de5b9ca6-caf7-43c8-a820-5540ec6df9b2",
+            "my_attribute",
+            "my_value",
+            {},
+            {
+                "data": {
+                    "type": "my_resource",
+                    "attributes": {
+                        "organization_id": "de5b9ca6-caf7-43c8-a820-5540ec6df9b2",
+                        "my_attribute": "my_value",
+                    },
+                },
+            },
+            id="no kwargs",
+        ),
+        pytest.param(
+            "my_resource",
+            "de5b9ca6-caf7-43c8-a820-5540ec6df9b2",
+            "my_attribute",
+            "my_value",
```

Comment:
Maybe just a nested value as a separate parametrized test for completeness?

Reply:
done!